execute/exectypes/outcome.go

```diff
@@ -77,12 +74,12 @@ func NewOutcome(
 	selectedCommits []CommitData,
 	report cciptypes.ExecutePluginReport,
 ) Outcome {
-	return newSortedOutcome(state, selectedCommits, report)
+	return NewSortedOutcome(state, selectedCommits, report)
 }
 
-// newSortedOutcome ensures canonical ordering of the outcome.
+// NewSortedOutcome ensures canonical ordering of the outcome.
 // TODO: handle canonicalization in the encoder.
```

Comment:
I probably put this TODO here. You made an encoder, I think it makes sense to move the sort there.